### PR TITLE
Updated Quick Start section on README

### PR DESCRIPTION
**Pull request by nkashy1**
In response to [Issue 91](https://github.com/google/kubeflow/issues/91),
raised by @sebgoa.

Although I was the one who had originally complained about the ksonnet
installation, I think the problem has been solved by not asking the user
to manually install it from source anymore and it's okay to point at the
top-level of the ksonnet documentation rather than the release page.

### CODE REVIEW

**Comment by nkashy1** [comment]
@sebgoa , @jlewi : Please have a look

**Comment by sebgoa** [comment]
sure, at least it does not send me to the release page that does not have the OSX bin.

**Comment by nkashy1** [comment]
@jlewi : Could you merge? I do not have sufficient permissions.

**Comment by jlewi** [comment]
@nkashy1 Looks like it got out of date. Can you resolve conflicts?

File: README.md [hunk 1]
```diff
@@ -44,11 +44,13 @@ If you want to use GPUs be sure to follow the Kubernetes [instructions for enabl
 
 ## Quick Start
 
-Requirements
+### Requirements
 
-  * ksonnet version [0.8.0](https://github.com/ksonnet/ksonnet/releases) or later.
+  * ksonnet version [0.8.0](https://github.com/ksonnet/ksonnet) or later.
```
Comment (jlewi) [inline README.md]: I think https://ksonnet.io/#get-started is a better link.

Comment (nkashy1) [reply README.md]: Done

**Pull request merged**